js/languages/pl.json

```diff
@@ -1,410 +0,0 @@
-{
```

Comment:
I think this could stay. A small number of the countries are actually translated and it's useful for testing purposes to have them there.